inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/consts/AttributeConstants.java

```diff
@@ -88,6 +88,8 @@ public class AttributeConstants {
 
     public static final String MESSAGE_SYNC_SEND = "syncSend";
 
+    public static final String MESSAGE_PROXY_SEND = "proxySend";
```

Comment:
It's better move these to common module, they are also used by`dataproxy-sdk`

Reply:
ok, you're right!